InternetStoreCHIReloaded/API/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using API.ViewModels;
+using AutoMapper;
+using BLL.Contracts;
+using BLL.Interfaces;
+using DAL.Entities;
+using DAL.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductsController : ControllerBase
+    {
+        private readonly IProductsService _productService;
+        private readonly IMapper _mapper;
+
+        public ProductsController(IProductsService productService, IMapper mapper)
+        {
+            _productService = productService;
+            _mapper = mapper;
+        }
+
+        [HttpGet]
+        public async Task<GenericResponse<IEnumerable<ProductViewModel>>> Get()// return all, add pagination next
```

Comment:
It's helpful for you to leave those comments now.
In live projects such simple comments are redundant because REST principals are used and method responsibility is obvious.
Swagger description could be used for such type of method description.